bundled/README.md

```diff
@@ -1,64 +1,54 @@
 This folder contains third party projects bundled with deal.II
+==============================================================
 
-PLEASE NOTE THAT THESE PROJECTS ARE COPYRIGHTED BY OTHERS THAN THE deal.II
+**PLEASE NOTE THAT THESE PROJECTS ARE COPYRIGHTED BY OTHERS THAN THE deal.II
 AUTHORS, but are included by permission. For details, consult the stated
-licenses below.
+licenses below.**
```

Comment:
This paragraph has a slightly ODD mixture OF lower AND upper CASE spelling. I know why you do it, it just looks odd. I guess it's not that big a deal, though.